net/pfSense-pkg-WireGuard/files/usr/local/pkg/wireguard/wg_globals.inc

```diff
@@ -118,6 +133,34 @@ function array_unset_value(&$array, $parents) {
 
 }
 
+/*
+ * This is a generalized implementation of init_config_arr
```

Comment:
Out of curiosity, is it not possible to use `init_config_arr()`?

Reply:
I touch the wireguard XML config by way of $wgg['config'], $wgg['tunnels'], and $wgg['peers'] which these reference $config in wg_globals.inc. Instead of initializing when necessary through $config, generalizing init_config_arr() allows me to perform the same action but on an arbitrary target (i.e. one of the $wgg targets above).